package.json

```diff
@@ -57,8 +57,8 @@
     }
   },
   "lint-staged": {
-    "*.js": [
-      "./node_modules/.bin/eslint"
+    "*.{js,ts}": [
+      "./node_modules/.bin/prettier --write", "./node_modules/.bin/eslint"
```

Comment:
i don't think we want prettier to write our changes when we're about to commit, since that introduces changes the developer didn't have time to review?

instead, we should introduce a `.prettierrc` file that defines our rules, and then each developer can set up their VScode to format on save so they can see their changes before committing

Reply:
A prettierrc.json file which handles that already exist in the project, prettier only reformats the code during commit to ensure all committed codes use the same formatting. I thought you wanted to enforce a consistent code style throughout the project, maybe I misunderstood the issue.